Resources/Prototypes/_NF/Entities/Objects/Fun/toys.yml

```diff
@@ -527,4 +527,41 @@
       size: Ginormous
       sprite: Objects/Weapons/Melee/Throngler-in-hand.rsi
     - type: DisarmMalus
-      malus: 0
\ No newline at end of file
+      malus: 0
+
+- type: entity
+  name: Emotional support RPG
+  parent: WeaponLauncherRocket
+  id: EmotionalsupportRPG
+  suffix: Toy
+  description: A deactive RPG modified to only allow Foam RPG warheads. "When in times of trouble and stress out as can be, DONT worry now because S.E.S.W.C has what you need!"
```

Comment:
Nitpicks:
* _A deactive_ → _A deactivated_ or _An inactive_
* _stress out_ → _stressed out_
* _DONT_ → _DON'T_ (apostrophe!)

Moreover, a question: How come 'Foam' is capitalised? Is 'Foam RPG' meant to be a brand name? 'Foam' is not otherwise a proper noun.

Reply:
the foam is capitalized is because it's just how I type. Writing is like that also for me

Reply:
fix